lib/jgi_mg_assembly/runner/pipeline.py

```diff
@@ -13,7 +13,6 @@
 
 PIGZ = "pigz"
 
-
 class Pipeline(object):
     def __init__(self, callback_url, scratch_dir):
```

Comment:
Inside here, I'm seeing FileUtil referenced, but I'm not seeing it imported anywhere?

Reply:
same for subprocess, used further down in this file

Reply:
oops, fixed